src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -289,6 +294,32 @@ def _send_verification_code_to_user(
     )
 
 
+def _send_privacy_request_receipt_email_to_user(
+    db: Session, policy: Policy, email: Optional[str]
+) -> None:
+    """Helper function to send request receipt email to the user"""
+    if not email:
```

Comment:
If this function will error if no `email` is provided, should the function be optional?

Reply:
I'd prefer to keep `email` optional here so that this helper function can encapsulate handling err cases for params needed to send an email. Is that what you meant?